### PR TITLE
fix(policyyaml) fix policy yaml backendref tags

### PR DESCRIPTION
**Pull request by slonka**
Fix backendref with tags and mirror

Did you sign your commit? [Instructions](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md#sign-your-commits) yes

Have you read [Contributing guidelines](https://github.com/kumahq/.github/blob/main/CONTRIBUTING.md)? yes

### CODE REVIEW

File: spec/fixtures/mtr.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+        _port: 3001
+      rules:
+        - default:
+            backendRefs:
+              - kind: MeshService
+                name: backend
+                _version: v1
+                namespace: kuma-demo
+                _port: 3001
+                port: 3001
```
Comment (Automaat) [inline spec/fixtures/mtr.yaml:23]: do we need `port` in this example?

File: spec/fixtures/mhr-port.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+                  name: mirror-this-request
+                  value: "true"
+          default:
+            filters:
+              - type: RequestMirror
+                requestMirror:
+                  percentage: 30
+                  backendRef:
+                    kind: MeshService
+                    name: backend_v1_experimental
```
Comment (Automaat) [inline spec/fixtures/mhr-port.yaml:29]: i think this should use `_version` instead of full name

**Comment by slonka** [comment]
[localhost:8000.zip](https://github.com/user-attachments/files/17588078/localhost.8000.zip)

```
python -m http.server 8000
```

open http://localhost:8000/localhost%3A8000/spec/fixtures/mhr-and-mtr.golden.html

**Comment by slonka** [comment]
new rendered:

[localhost:8000.zip](https://github.com/user-attachments/files/17588208/localhost.8000.zip)

File: spec/fixtures/mtr.yaml (hunk 1 above)
Comment (slonka) [reply spec/fixtures/mtr.yaml:22]: we only need `port`, not `_port`

Comment (Automaat) [reply spec/fixtures/mtr.yaml:22]: so we can remove it from this example

Comment (Automaat) [reply spec/fixtures/mtr.yaml:22]: so we now don't use _port in our examples? won't it break all of the rest already migrated policies?

Comment (slonka) [reply spec/fixtures/mtr.yaml:22]: `_port` in `targetRef` is different than `port` in `backendRef`

**Review by Automaat — APPROVED** [review]

**Pull request merged**